src/watson.ts

```diff
@@ -0,0 +1,824 @@
+          args: {
+            name: "DATETIME",
+            description: "The start time. Ex: 2022-03-04 12:00:00",
+          },
+        },
+        {
+          name: ["--to", "-t"],
+          description: "Date and time of end of tracked activity",
+          args: {
+            name: "DATETIME",
+            description: "The start time. Ex: 2022-03-04 12:00:00",
+          },
+        },
+        {
+          name: ["--confirm-new-project", "-c"],
+          description: "Confirm creation of new tag",
+        },
+      ],
+    },
+    {
+      name: "aggregate",
+      description:
+        "Display a report of the time spent on each project aggregated by day",
+      options: [
+        {
+          name: ["--current", "-c"],
+          description: "Include currently running frame in report",
+          exclusiveOn: ["--no-current", "-C"],
+        },
+        {
+          name: ["--no-current", "-C"],
+          description: "Don't include currently running frame in report",
+          exclusiveOn: ["--current", "-c"],
+        },
+        {
+          name: ["--from", "-f"],
+          description:
+            "The date from when the reports should start. Defaults to seven days ago",
+          args: {
+            name: "DATETIME",
+            description: "The start time. Ex: 2022-03-04 12:00:00",
+          },
+        },
+        {
+          name: ["--to", "-t"],
+          description:
+            "The date at which the report should stop (inclusive). Defaults to tomorrow",
+          args: {
+            name: "DATETIME",
+            description: "The start time. Ex: 2022-03-04 12:00:00",
+          },
+        },
+        {
+          name: ["--project", "-p"],
+          description:
+            "Reports activity only for the given project. You can add other projects by using this option several times",
+          args: {
+            generators: listProjects,
+            name: "PROJECT",
+          },
+        },
+        {
+          name: ["--tag", "-T"],
+          description:
+            "Reports activity only for frames containing the given tag. You can add several tags by using this option multiple times",
+          args: {
+            generators: listTags,
+            name: "TAG",
+          },
+        },
+        {
+          name: ["--json", "-j"],
+          description: "Format output in JSON instead of plain text",
+          exclusiveOn: ["--csv", "-s"],
+        },
+        {
+          name: ["--csv", "-s"],
+          description: "Format output in CSV instead of plain text",
+          exclusiveOn: ["--json", "-j"],
+        },
+        {
+          name: ["--pager", "-g"],
+          description: "View output through a pager",
+          exclusiveOn: ["--no-pager", "-G"],
+        },
+        {
+          name: ["--no-pager", "-G"],
+          description: "Don't view output through a pager",
+          exclusiveOn: ["--pager", "-g"],
+        },
```

Comment:
These are all mutually exclusive:
```suggestion
        {
          name: ["--json", "-j"],
          description: "Format output in JSON instead of plain text",
          exclusiveOn: ["--csv", "--pager", "--no-pager"],
        },
        {
          name: ["--csv", "-s"],
          description: "Format output in CSV instead of plain text",
          exclusiveOn: ["--json", "--pager", "--no-pager"],
        },
        {
          name: ["--pager", "-g"],
          description: "View output through a pager",
          exclusiveOn: ["--csv", "--json", "--no-pager"],
        },
        {
          name: ["--no-pager", "-G"],
          description: "Don't view output through a pager",
          exclusiveOn: ["--csv", "--pager", "--json"],
        },
```